### PR TITLE
Update the menu tree editor in Reports

### PR DESCRIPTION
**Pull request by mfalesni**
{{pytest: cfme/tests/intelligence/reports/test_menus.py -v -k shuffle --long-running }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 6ea5bb664420b46e2b48a8756d5bde4f0eb8ede5:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 943ccb79f4c207096f212e5eaef93a518e7c767d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/intelligence/reports/ui_elements.py [hunk 1]
```diff
@@ -646,24 +651,45 @@ def bail_out(cls):
...
     # Button clicking functions
     def move_top(self):
-        self._click_button("Move selected folder top")
+        if version.current_version() < "5.5.0.7":
+            self._click_button("Move selected folder top")
+        else:
+            self._click_button_i("fa-angle-double-up")
 
     def move_bottom(self):
-        self._click_button("Move selected folder to bottom")
```
Comment (psav) [inline cfme/intelligence/reports/ui_elements.py]:
```python
def _fn(name, icon):
    if version.current_version() < "5.5.0.7":
        self._click_button(name)
    else:
        self._click_button_i(icon)
move_up = partial(_fn, "Move selected folder up", "fa-angle-up")

**Comment by dajoRH** [comment]
Lint report for commit 0582f60e46661af99f911ae8e1d40a1513324321:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cad68d4bb1be2dd92eba80e625d4af41ccf6dc45:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**